docs/spec_draft.md

```diff
@@ -62,6 +62,11 @@ At the moment, StableHLO only supports dense tensors, so each tensor has
     0 elements. Such tensors are allowed and are useful in rare cases, e.g.
     to model empty slices.
 
+**Function types** referred in the document as `function` type, represent
+functions that take 0 or more input arguments of any supported types, have 0 or
+more operations/statements in function body and return 0 or more results of any
+supported types.
```

Comment:
Rebase artifact? This has already been merged into `main`, so this diff shouldn't be here.

Reply:
Rebased and removed this from my changes